Source/CsDebugScript.DwarfSymbolProvider/ElfCoreDumpDebuggingEngine.cs

```diff
@@ -604,6 +604,37 @@ public void Terminate(Process process)
         {
             throw new NotImplementedException();
         }
+
+        /// <summary>
+        /// Adds new breakpoint to the given process.
+        /// </summary>
+        /// <param name="process">Process.</param>
+        /// <param name="address">Expression to be evaluated into breakpoint.</param>
+        /// <returns>New breakpoint.</returns>
+        public IBreakpoint AddBreakpointCode(Process process, ulong address)
+        {
+            throw new NotImplementedException();
+        }
+
+        /// <summary>
+        /// Adds new breakpoint with assosiated action.
+        /// </summary>
+        /// <param name="process">Process.</param>
+        /// <param name="breakpointSpec">Description of this breakpoint.</param>
+        /// <returns>New breakpoint.</returns>
+        public IBreakpoint AddBreakpoint(Process process, BreakpointSpec breakpointSpec)
+        {
+            throw new NotImplementedException();
+        }
+
+        /// <summary>
+        /// Returns address of requested symbol.
+        /// </summary>
+        /// <param name="symbol">The symbol.</param>
+        public ulong GetSymbolAddress(string symbol)
```

Comment:
>GetSymbolAddress [](start = 21, length = 16)

What is symbol? Function? UserType? Global variable? Constant? Try to be more specific. Some of the symbols don't have address...

Reply:
I am removing this code.